src/test/java/edu/ksu/canvas/attendance/services/AttendanceSummaryCSVServiceUTest.java

```diff
@@ -51,7 +55,9 @@
     private final int ARBITRARY_STUDENT_TOBE_MADEUP_3 = 6;
     private final int ARBITRARY_STUDENT_MIN_MISSED_3 = 100;
     private final int ARBITRARY_STUDENT_EXCUSED_3 = 2;
+    private final int ARBITRARY_STUDENT_PRESENT_3 = 2;
     private final double ARBITRARY_STUDENT_PERCENT_3 = 50.1;
+    private final String ARBITRARY_SIS_USER_ID_3 = "SIS_ID_3";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "ARBITRARY_SIS_USER_ID_3" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00116)